From 4449246a8d788e71dfe4679fba95955cec5894c1 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 3 Nov 2020 22:58:26 -0800
Subject: [PATCH] gst_private.h: increse padding in struct _GstClockEntryImpl

When compiling for 32bit architectures with 64bit time_t e.g. riscv32,
the static assert that the GstClockEntryImpl smaller or
equal to the struct _GstClockEntryImpl triggered.
(they were 12bytes off).

To fix this, the padding is increased by 8 bytes (on 32bit).

Upstream-Status: Submitted [https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/694]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 gst/gst_private.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/gst_private.h b/gst/gst_private.h
index a8897e1..8252ede 100644
--- a/gst/gst_private.h
+++ b/gst/gst_private.h
@@ -522,7 +522,7 @@ struct _GstClockEntryImpl
   GstClockEntry entry;
   GWeakRef clock;
   GDestroyNotify destroy_entry;
-  gpointer padding[19];                 /* padding for allowing e.g. systemclock
+  gpointer padding[21];                 /* padding for allowing e.g. systemclock
                                          * to add data in lieu of overridable
                                          * virtual functions on the clock */
 };
-- 
2.29.2

